app/components/provider_interface/user_permission_summary_component.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= tag.dl(class: 'govuk-summary-list') do %>
+  <% ProviderPermissions::VALID_PERMISSIONS.each do |permission| %>
+    <%= tag.div(class: 'govuk-summary-list__row') do %>
+      <dt class="govuk-summary-list__key">
+        <%= PERMISSIONS[permission] %>
+      </dt>
+      <dd class="govuk-summary-list__value">
```

Comment:
Is there a way we can use the existing summary list component now?

Reply:
No. I tried that approach initialy, there are a number of issues with rendering another component and then passing it in as part of the `rows` along with additional html. 

A way around it would be to amend the existing `SummaryList` component to accept a block but we discussed that with Abeer and seemed a bit excessive as we've not had that need for anything else using it.

Reply:
Ah, we actually did have to do this for the organisation_permissions_review_card_component.rb. The component wasn't as complex as this one, but I had to do `html_safe` to get it to render properly.

Maybe it would be worth doing but as a separate card?

Reply:
I did use safe_html. The issue was related to using the `render` method, clashing with rspec's render and then another error about `variants` which we were unable to pinpoint. It did work, but the specs were problematic. We ended up using ActionController::Base.render to work around the component's spec error but then the system specs started failing and we decided for a more straightforward way of doing this. It was all done on the same branch though so I don't have a copy of it.

I think it's really low priority atm so not sure there is much value in adding a card given how many bugs have been sitting for weeks. We did discuss a more general component review in the dev retro a couple of weeks ago so If anything we can try and identify if we need to improve the existing SummaryList as part of that.